src/main/webapp/user-page.html

```diff
@@ -46,8 +48,6 @@ <h1 id="page-title">User Page</h1>
       <input type="submit" value="Submit">
     </form>
     <hr/>
-
```

Comment:
These three lines don't need to change, so you can leave them the way they were.